site/update-image.md

```diff
@@ -62,6 +62,12 @@ The input for this parameter is a simple text file that contains one or more of
 
 Each section can contain one or more valid Dockerfile commands and would look like the following:
 
+#### `--latestPSU`
+
+The `latestPSU` option will continue to be supported for the CREATE option, but has been deprecated for use in the 
+UPDATE option.  Because of the number of patches and the size, using `latestPSU` as an update to an existing image can 
```

Comment:
Because of the number of patches and the size, -> Because of the number of patches and their size,